docs/catalog/cloud/choose-sustainable-region.md

```diff
@@ -0,0 +1,44 @@
+---
+version: 1.0
+submitted_by: markus-ntt-seidl
+published_date: TBD
+category: cloud
+tags: 
+ - cloud
+ - role:cloud-engineer
+ - size:small
+---
+
+# Choose sustainable region for workloads
+
+## Description
+
+Applications can have background tasks or jobs that run independent to the rest of the application. These workloads can be put into a region that is the most sustainable region of the used cloud provider.
+
+## Solution
+
+Run the workloads in a region that the used cloud provider labels as most sustainable. Choose the region that is still near to your application so that data transfer to and from the sustainable region is minimal.
+
+## SCI Impact
+
+`SCI = (E * I) + M per R`  
+[Software Carbon Intensity Spec](https://grnsft.org/sci)
+
+Concerning the SCI equation. choose sustainable region for workloads will impact two parts:
+
```

Comment:
How about M? From the other pattern: [M: By reducing the total number of computing equipment traversed, the total embodied carbon is lower.](https://github.com/Green-Software-Foundation/green-software-patterns/blob/dev/docs/catalog/cloud/choose-region-closest-to-users.md)

Reply:
I agree with @greenhsu123 that this would also affect M.